src/server.c

```diff
@@ -5926,6 +5941,7 @@ sds genValkeyInfoString(dict *section_dict, int all_sections, int everything) {
             "current_eviction_exceeded_time:%lld\r\n", current_eviction_exceeded_time / 1000,
             "keyspace_hits:%lld\r\n", server.stat_keyspace_hits,
             "keyspace_misses:%lld\r\n", server.stat_keyspace_misses,
+            "keyspace_hit_ratio:%2.f%%\r\n", getCacheHitRatioMetric(server.stat_keyspace_hits, server.stat_keyspace_misses),
```

Comment:
Can I ask why you aren't able to compute this yourself on the client side? The reason we generally avoid these types of server side floating aggregate metrics is that they take some amount of CPU time for everyone and you can just compute it yourself.

Reply:
Thank you for your feedback.
@madolson, I agree with you.

Everyone has different CPU times. However, I thought that if I provided a keyspace hit ratio based on the keyspace_hits and keyspace_misses metrics, I could devise a cache strategy that could improve system performance.
Ex) Cache strategy (data characteristics, application requirements and etc)

This can be said for data cache usage, but for session usage, the cache hit rate may be low.

Also, the user can calculate it himself, but if Valkey's info stats shows the keyspace_hit_ratio metric with keyspace_hits and keyspace_misses, Because of this, I think that the cache expire period(time) can be set efficiently to increase the Valkey cache hit ratio.
In addition to floating metrics, integer metrics can also be considered.

However, I think the performance of valkey comes first and is important.

Reply:
@Virusuki Would you mind opening an issue for this and linking this PR? I think what we'll do is close this and see if anyone else else comments on the issue. If other members of the community are interested, we can always revive this PR and merge it at a later point. Sound reasonable?

Reply:
@madolson thanks for your comment. 😄